en/core-utility-libraries/hash.rst

```diff
@@ -73,6 +73,32 @@ Attribute Matching Types
 |                                | the regular expression inside ``...``.     |
 +--------------------------------+--------------------------------------------+
 
+    Use matchers by appending them to the expression element (``{n}``, ``{s}``, etc.) you wish to match.
+    
+    So to return ``id`` fields where a ``name`` matches you can use paths using ``{n}`` and ``{s}`` to insert data into multiple
+    points::
+
+        $users = Array(
+             Array(
```

Comment:
Use short array syntax `[ ]`

Reply:
Yep - I would have - but it's not consistent with the rest of the page then...

Reply:
I thought `master` is for `3.0` docs https://book.cakephp.org/3.0/en/core-libraries/hash.html

2.0 uses `Array()` https://book.cakephp.org/2.0/en/core-utility-libraries/hash.html, so format is then good.